drivers/video/drm/rockchip_display.c

```diff
@@ -1457,7 +1458,14 @@ static int load_bmp_logo(struct logo_info *logo, const char *bmp_name)
 
 	bmp_create(&bmp, &bitmap_callbacks);
 
+#ifdef CONFIG_RADXA_IMG
+	len = radxa_read_bmp_file(bmp_data, bmp_name);
+#elif CONFIG_ROCKCHIP_RESOURCE_IMAGE
 	len = rockchip_read_resource_file(bmp_data, bmp_name, 0, MAX_IMAGE_BYTES);
+#else
+	return -EINVAL;
+#endif
```

Comment:
```suggestion
#ifdef CONFIG_RADXA_IMG
	len = radxa_read_bmp_file(bmp_data, bmp_name);
#else
	len = rockchip_read_resource_file(bmp_data, bmp_name, 0, MAX_IMAGE_BYTES);
#endif
```
更好的是在radxa_read_bmp_file调用后如果失败的话，再调用rockchip_read_resource_file